scala/README.md

```diff
@@ -145,7 +156,7 @@ Please read the [Scala Style Guide] carefully. The main points to consider are:
 
     Inside each group, packages and classes must be sorted alphabetically. Separate groups with blank lines.
 
-    Use the following mnemonic to remember package order: your project is built on top of the frameworks, which is built on Scala, which is built on top of Java. Third-party libraries are less important, therefore come last.
+    Use the following mnemonic to remember package order: your project is built on top of the frameworks, which are built on Scala, which is built on top of Java. Third-party libraries are less important, therefore come last.
```

Comment:
As long as it is documented, makes sense, is easy to remember, and is followed by all projects, I'm fine with any order.